app/src/main/java/com/infomaniak/mail/ui/main/folder/ThreadListFragment.kt

```diff
@@ -337,6 +338,7 @@ class ThreadListFragment : TwoPaneFragment(), SwipeRefreshLayout.OnRefreshListen
         }
 
         searchButton.setOnClickListener {
+            mainViewModel.isInSearch.value = true
             safeNavigate(
                 ThreadListFragmentDirections.actionThreadListFragmentToSearchFragment(
                     dummyFolderId = mainViewModel.currentFolderId ?: "eJzz9HPyjwAABGYBgQ--", // Hardcoded INBOX folder
```

Comment:
What about putting this inbox folder ID in the companion of the FolderController, along with the searchFolderId ?